code/controllers/subsystem/uniqueness.dm

```diff
@@ -0,0 +1,119 @@
+//* This file is explicitly licensed under the MIT license. *//
+//* Copyright (c) 2025 Citadel Station Developers           *//
+
+/**
+ * Nebula calls this a repository, I call this a subsystem.
+ * This is in charge of assigning non-duplicating IDs to things.
+ * * Also in charge of round and map mangling.
+ */
+SUBSYSTEM_DEF(uniqueness)
```

Comment:
SSmapping has an obfuscation module but it doesn't really belong in mapping. We should move things over here as we go.